modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -298,9 +302,10 @@ enum
     CV_CAP_PROP_XI_TARGET_TEMP                                  = 467, // Set sensor target temperature for cooling.
     CV_CAP_PROP_XI_CHIP_TEMP                                    = 468, // Camera sensor temperature
     CV_CAP_PROP_XI_HOUS_TEMP                                    = 469, // Camera housing tepmerature
+    CV_CAP_PROP_XI_HOUS_BACK_SIDE_TEMP                          = 590, // Camera housing back side tepmerature
     CV_CAP_PROP_XI_CMS                                          = 470, // Mode of color management system.
     CV_CAP_PROP_XI_APPLY_CMS                                    = 471, // Enable applying of CMS profiles to xiGetImage (see XI_PRM_INPUT_CMS_PROFILE, XI_PRM_OUTPUT_CMS_PROFILE).
-    CV_CAP_PROP_XI_IMAGE_IS_COLOR                               = 474, // Returns 1 for color cameras.
+    CV_CAP_PROP_XI_IMAGE_IS_COLOR                               = 473, // Returns 1 for color cameras.
```

Comment:
Enum value is changed here (ABI violation issue). Is it a bug fix?

List of changed/removed enum values:
http://pullrequest.opencv.org/buildbot/builders/precommit_linux64/builds/5314/steps/Compare%20ABI%20dumps/logs/report-html#Constant_Source_Problems_Low